templates/ui-package/src/app/auth/auth.component.ts

```diff
@@ -29,18 +26,12 @@ export class AuthComponent implements OnInit {
   }
 
   ngOnInit() {
-    // Observable for receiving the auth's events, also you can use the default behaviour
-    // which is redirect
-    this.authService.onAuthorizationResult().subscribe(result => {
-      if (result.authorizationState === 'authorized') {
-        console.log(\`authorized\`);
-
-      }
-    });
-    //Observable to determine if is authorized or not
+    // Observable to determine if is authorized or not, default value is undefined
     this.authService.isAuthorized().subscribe(authorized => {
-      this.isLoggingEnable =authorized;
-      console.log('is authorized: ' + authorized);
+      if(authorized || authorized === false) {
```

Comment:
What are the possibilities for authorized variable?

Reply:
true  false or undefined